test/browser/shared/bootstrap.js

```diff
@@ -2,7 +2,6 @@ const {promisify} = require('util')
 const mkdirp = promisify(require('mkdirp'))
 const path = require('path')
 const puppeteer = require('puppeteer')
-const test = require('tape-catch')
```

Comment:
Why remove the `tape-catch`?

Reply:
It is not used in this file, so eslint was complaining about it.